go/adbc/pkg/_tmpl/driver.go.tmpl

```diff
@@ -19,7 +19,11 @@
 
 package main
 
-// #cgo CXXFLAGS: -std=c++11
+// ADBC_EXPORTING is required on Windows, or else the symbols
+// won't be accessible to the driver manager
+
+// #cgo CFLAGS: -DADBC_EXPORTING
```

Comment:
Should we make the ADBC_EXPORTING only get defined on Windows? Or have it defined for all platforms?